benefits/core/views.py

```diff
@@ -43,7 +43,10 @@ def index(request):
         classes="home",
     )
 
-    return TemplateResponse(request, TEMPLATE_PAGE, page.context_dict())
+    help_page = reverse(ROUTE_HELP)
+    context_dict = {**page.context_dict(), **{"info_link": f"{help_page}#about"}}
+
+    return TemplateResponse(request, TEMPLATE_PAGE, context_dict)
```

Comment:
I think this change might've been put here on accident(?) and is not needed. `info_link` is not actually used here on the core index page.